lib/runc_sandbox.ml

```diff
@@ -5,8 +5,26 @@ let ( / ) = Filename.concat
 type t = {
   runc_state_dir : string;
   fast_sync : bool;
+  arches : string list;
 }
 
+let get_machine () =
+  let ch = Unix.open_process_in "uname -m" in
+  let arch = input_line ch in
+  match Unix.close_process_in ch with
+  | Unix.WEXITED 0 -> String.trim arch
+  | _ -> failwith "Failed to get arch with 'uname -m'"
+
+let get_arches () =
+  if Sys.unix then (
+    match get_machine () with
+    | "x86_64" -> ["SCMP_ARCH_X86_64"; "SCMP_ARCH_X86"; "SCMP_ARCH_X32"]
+    | "aarch64" -> ["SCMP_ARCH_AARCH64"; "SCMP_ARCH_ARM"]
+    | _ -> []
```

Comment:
could we enumerate this somehow so that it'll fail on an unknown arch?  Otherwise we'll run into this when adding riscv-32 in the future. (or could just make a note to remember to update this somewhere when we get around to riscv32)

Reply:
there is logic in https://github.com/avsm/osrelease/blob/master/lib/osrelease.ml that i could release that does all the arch detection (based on opams), if that helps

Reply:
Could do. But when we add a new multi-arch platform then we'll test it and discover the problem immediately anyway.